src/lottery/index.js

```diff
@@ -27,15 +22,19 @@ lotteryRouter.use("/global-state", require("./routes/globalState"));
 lotteryRouter.use("/transactions", require("./routes/transactions"));
 lotteryRouter.use("/items", require("./routes/items"));
 
-const eventStatusResource = buildResource([instagramRewardAction])(
-  eventStatusModel
-);
-const otherResources = [eventModel, itemModel, transactionModel].map(
-  buildResource()
-);
+const resources = [
+  eventStatusModel,
+  eventModel,
+  itemModel,
+  transactionModel,
+].map(buildResource());
+
+const contracts = eventMode
+  ? require(`./modules/contracts/${eventMode}`)
+  : undefined;
```

Comment:
`eventMode` `type`이 `string | undefined` 라면

```suggestion
const constracts = eventMode && require("./module/constracts/${eventMode}");
```

로 바뀌어도 될 것 같군요

Reply:
더 짧고 좋네요! 그런데 `constracts` 가 아니라 `contracts` 라서.. ㅠ 제가 로컬에서 직접 수정해서 커밋하도록 하겠습니다. 좋은 의견 감사합니다!